irksome/tools.py

```diff
@@ -9,6 +9,9 @@
 from irksome.deriv import TimeDerivative
 
 
+vecconst = numpy.vectorize(Constant)
```

Comment:
This doesn't seem to be being used, but duplicates a line from `stage.py`.  Am I missing something or is this truly orphaned?

Reply:
At some point I was planning to stick `vecconst` in `tools.py` but this line is currently orphaned.